gulpfile.js

```diff
@@ -157,6 +161,8 @@ createTask('process-github-issues', processGithubIssues);
 createTask('serve', serve);
 createTask('server-tests', serverTests);
 createTask('size', size);
+createTask('storybook:amp', storybookAmp);
+createTask('storybook:preact', storybookPreact);
```

Comment:
Nit: We've been moving away from colons in task names (other than a couple of legacy names that  still use them). Can you rename these to `storybook-amp` and `storybook-preact`?